src/views/download-view.js

```diff
@@ -164,7 +164,7 @@ class DownloadView extends ReduxMixin(PolymerElement) {
                 </p>
             </div>
             
-            <template is="dom-if" if="[[isSelectedVersion3XX]]">
```

Comment:
By removing this part, the message box seems to be no longer displayed.
It should be visible.